xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -128,7 +128,7 @@ Rail transport, Roads and Road transport
 		</xsd:sequence>
 	</xsd:group>
 	<!-- ====ACCESS=================================================================== -->
-	<xsd:element name="AccessEquipment" type="AccessEquipment_VersionStructure" substitutionGroup="InstalledEquipment">
+	<xsd:element name="AccessEquipment" type="AccessEquipment_VersionStructure" abstract="true" substitutionGroup="InstalledEquipment">
```

Comment:
What does this 'abstract' mean?

Reply:
^
 |

Reply:
Before fix:

![AccessEquipment-list-element](https://user-images.githubusercontent.com/18263655/116146003-483f7180-a6de-11eb-84d3-1e4299617109.png)

=> Validation error "The type definition cannot be abstract for element AccessEquipment."